src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/repository/CardRepository.java

```diff
@@ -8,10 +8,17 @@
 import org.springframework.data.repository.query.Param;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 import org.springframework.data.rest.core.annotation.RestResource;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.parameters.P;
 
 @RepositoryRestResource(excerptProjection = CardEmbedProjection.class)
 public interface CardRepository extends JpaRepository<Card, Long> {
   @RestResource(exported = false)
+  @Override
+  @SuppressWarnings({"SpringElInspection", "ELValidationInspection"})
+  @PreAuthorize("hasRole('PLAYER') and isOwnCard(#card)")
+  <S extends Card> S save(@P("card") S entity);
+
```

Comment:
```suggestion

  @RestResource(exported = false)
```
But you still need it here